src/task-model.js

```diff
@@ -0,0 +1,24 @@
+function Task(name, description, date, priority, status = false) {
+  this.id = Date.now();
+  this.name = name;
+  this.description = description;
+  this.date = date;
+  this.priority = priority;
+  this.status = status;
+}
+
+// class Task {
+//   constructor(id, name, description, date, priority, status = false) {
+// /*eslint-disable */
+//     /* eslint-enable */
+// this.id = id;
+// this.name = name;
+// this.description = description;
+// this.date = date;
+// this.priority = priority;
+// this.status = status;
+//   }
+// }
```

Comment:
- [ ] [REQUIRED] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))